snap/snapcraft.yaml

```diff
@@ -0,0 +1,82 @@
+name: xmrig
+adopt-info: xmrig
+summary: XMRig is a high performance, open source, cross platform CPU/GPU miner
+description: |
+  XMRig is a high performance, open source, cross platform RandomX, KawPow, 
+  CryptoNight and AstroBWT unified CPU/GPU miner and RandomX benchmark. 
+  
+  To run, open a terminal:
+    - Ensure you've identified the options you'd like to run
+    - Identify the pool (and port) desired
+    - Use your wallet address
+      - Example: `xmrig -o example.pool.com:4444 -u [wallet address]`
+    - Hit enter and profit
+    
+license: GPL-3.0
+
+base: core20
+grade: stable
+confinement: strict
+compression: lzo
+
+architectures:
+  - build-on: amd64
+    run-on: [amd64, armhf, arm64, ppc64el, s390x]
```

Comment:
Technically could have `build-on: [amd64, arm64]`

Seems like run-on should have the same list as build-on, ppc64el and s390x definitely unsupported, etc.

Reply:
This is snapcraft formatting. I am checking into it's capability to run on other arches.

Reply:
Okay, reading this [documentation](https://snapcraft.io/docs/architectures), I will make sure the two architectures you noted above are the only ones; that's assuming that I'm interpreting the language appropriately.